Symfony/CS/Fixer/Contrib/EregToPregFixer.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/*
+ * This file is part of the Symfony CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Matteo Beccati <matteo@beccati.com>
+ */
+class EregToPregFixer extends AbstractFixer
+{
+    /**
+     * @var array
+     */
+    private $map = array(
```

Comment:
This array is not a map (a map is an associative array). So the naming is quite bad.